cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectClusterTest.java

```diff
@@ -1282,10 +1282,10 @@ public void testDefaultPodDisruptionBudget() {
     public void testImagePullPolicy() {
         KafkaConnectCluster kc = KafkaConnectCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, resource, VERSIONS, SHARED_ENV_PROVIDER);
 
-        Deployment dep = kc.generateDeployment(replicas, null, Collections.EMPTY_MAP, true, ImagePullPolicy.ALWAYS, null, null);
+        Deployment dep = kc.generateDeployment(replicas, null, emptyMap(), true, ImagePullPolicy.ALWAYS, null, null);
```

Comment:
I opted to stick with emptyMap() instead of Map.of() as it's already used in a lot of places. I just fixed the lines that uses the EMPTY_MAP field.